src/components/transport_manager/src/transport_manager_impl.cc

```diff
@@ -426,11 +426,13 @@ int TransportManagerImpl::SearchDevices() {
   return transport_adapter_search;
 }
 
-int TransportManagerImpl::Init() {
-  LOG4CXX_TRACE(logger_, "enter");
-  is_initialized_ = true;
-  LOG4CXX_TRACE(logger_, "exit with E_SUCCESS");
-  return E_SUCCESS;
+int TransportManagerImpl::Init(resumption::LastState &last_state) {
+    // Last state requred to initialize Transport adapters
+    UNUSED(last_state);
```

Comment:
@Kozoriz @LuxoftAKutsan I think it's a bad architecture. We have interface class, we have Impl class and we have default class for the TM. If you can't change it right now - please create task for changing it in future.